addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -88,4 +96,43 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
     this.hasBeenFocused = true;
     super.updateValidations();
   }
+
+  getFieldOptionsAsObject(fieldOptions) {
+    if (typeof fieldOptions == 'string') {
+      try {
+        return JSON.parse(fieldOptions.trim());
+      } catch (error) {
+        console.error(` | Could not parse string options to json object`);
+        console.error(` | `, error);
+
+        return {};
+      }
+    }
+
+    return fieldOptions;
+  }
+
+  isValidFieldOptions(fieldOptions) {
+    let config = this.getFieldOptionsAsObject(fieldOptions);
+
+    if (!config) {
+      return false;
+    }
+
+    const requiredProperties = ['conceptScheme'];
+    const missingProperties = [];
+    for (const required of requiredProperties) {
+      if (!Object.keys(config).includes(required)) {
+        missingProperties.push(required);
+      }
+    }
+
+    if (missingProperties.length !== 0) {
+      console.warn(`Object is missing keys: `, missingProperties.join(', '));
```

Comment:
It might be nice to also provide list the component name, or mention which form field has this issue? To make it easier to debug? It doesn't really matter for the form-builder use-case, but it might help for the regular semantic form setups I think.

Reply:
Added this one, would be nice indeed :D  
![image](https://github.com/lblod/ember-submission-form-fields/assets/36266973/99bb49a4-59ff-46c1-b6fd-5a88f5a1e35b)